Neo4j.Driver/Neo4j.Driver/Internal/Result/Summary/DatabaseInfo.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) "Neo4j"
+// Neo4j Sweden AB [https://neo4j.com]
+// 
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Neo4j.Driver.Internal.Result;
+
+internal class DatabaseInfo : IDatabaseInfo
+{
+    public DatabaseInfo()
+        : this(null)
+    {
+    }
+
+    public DatabaseInfo(string name)
+    {
+        Name = name;
+    }
+
+    public string Name { get; }
```

Comment:
Just a personal style comment really. I prefer properties and member variables to be at the top of the class before the methods for ease of parsing.

Reply:
fields > constructors > properties > methods is our coding standard